git_machete/utils.py

```diff
@@ -237,16 +239,16 @@ class AnsiEscapeCodes:
 
 
 def bold(s: str) -> str:
-    return s if ascii_only or not s else AnsiEscapeCodes.BOLD + s + AnsiEscapeCodes.ENDC
+    return s if ascii_only or not s else AnsiEscapeCodes.BOLD + s + AnsiEscapeCodes.ENDC_BOLD_DIM
```

Comment:
Nice, I didn't know there's a way to just turn off a single aspect of formatting via ANSI espaces 😯